librz/analysis/arch/ppc/ppc_il_ops.c

```diff
@@ -0,0 +1,325 @@
+// SPDX-FileCopyrightText: 2022 Rot127 <unisono@quyllur.org>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "ppc_il.h"
+#include "ppc_analysis.h"
+#include <rz_il/rz_il_opcodes.h>
+#include <rz_util/rz_assert.h>
+#include <capstone.h>
+#include <rz_il/rz_il_opbuilder_begin.h>
+
+/**
+ * \brief Handles all supported LOAD operations.
+ *
+ * \param handle The capstone handle.
+ * \param insn The capstone instruction.
+ * \param mode The capstone mode.
+ * \return RzILOpEffect* Sequence of effects.
+ */
+static RzILOpEffect *load_op(RZ_BORROW csh handle, RZ_BORROW cs_insn *insn, const cs_mode mode) {
+	rz_return_val_if_fail(handle && insn, NOP);
+	ut32 id = insn->id;
+	// READ
+	const char *rT = cs_reg_name(handle, INSOP(0).reg);
+	const char *rA = cs_reg_name(handle, INSOP(1).reg);
+	const char *rB = cs_reg_name(handle, INSOP(2).reg);
+	st64 d = INSOP(1).imm;
+	st64 sI = d;
+	bool update_ra = false; // Save ea in RA?
+	ut32 mem_acc_size = ppc_get_mem_acc_size(id);
+	RzILOpPure *op0;
+	RzILOpPure *op1;
+	RzILOpPure *ea;
+	RzILOpPure *into_rt;
+
+	if (!rA && id == PPC_INS_LI) {
+		// LLVM/capstone bug?
```

Comment:
Don't forget to report in the capstone repository, and add here a link to the issue.